### PR TITLE
Remove Databinding dependency

### PR DESCRIPTION
**Pull request by meierjan**
As requested in #227 I removed the data-binding library as a dependency and bound the view by hand.

I just run the example to test so please also have a look.

### CODE REVIEW

**Comment by heinrichreimer** [comment]
Travis fails to build, I can check that later today.
Seems to be some kind of issue with licenses.

**Comment by meierjan** [comment]
Yes, it is.

**Comment by meierjan** [comment]
Hey, just wanted to hear if you find time to review/merge this?

**Comment by heinrichreimer** [comment]
I'm on holidays. You'd need to export the constraint layout license from Android Studio into Travis CI as Travis seems to not support that license out of the box: http://d.android.com/r/studio-ui/export-licenses.html
After exporting the license hash you can import it into the .travis.yml file like this: https://github.com/heinrichreimer/material-intro/blob/master/.travis.yml#L15

**Comment by heinrichreimer** [comment]
The license issue of Travic CI not accepting the constraint layout license is discussed here: https://github.com/travis-ci/travis-ci/issues/6617
@meierjan Maybe you find time to implement a fix as described in that issue?
Again, once the license issue is fixed and Travis builds as it should, I'll merge the PR.

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ jdk:
   - oraclejdk8
 before_install:
   - mkdir "${ANDROID_HOME}/licenses" || true
+  - echo -e "\nd56f5187479451eabf01fb78af6dfcb131a6481e" > "${ANDROID_HOME}/licenses/android-sdk-license"
```
Comment (heinrichreimer) [inline .travis.yml]: That doesn't seem to work either. Maybe the license hash is wrong or maybe the license name?

Comment (meierjan) [reply .travis.yml]: yeah, I saw but I was pretty much out of ideas. It's pretty nasty to test too. I can just force push...

**Comment by meierjan** [comment]
there u go.

**Comment by meierjan** [comment]
Can we have a new relase without data-binding?

**Pull request merged**